src/components/dac_dataset_table/DACDatasetTableCellData.js

```diff
@@ -0,0 +1,143 @@
+import {DAC} from '../../libs/ajax';
+
+export const consoleTypes = {
+  CHAIR: 'chair',
+  DATA_SUBMITTER: 'dataSubmitter',
+};
+
+export function duosIdCellData({dataset, label='duosIdCellData'}) {
+  return {
+    data: div({
+      style: {
+        display: 'flex',
+        alignItems: 'center',
+      }
+    }, [h(Link, {to: `dataset_statistics/${dataset.dataSetId}`}, [dataset.alias])]),
+    value: dataset.alias,
+    id: dataset.dataSetId,
+    cellStyle: { width: styles.cellWidths.duosId },
+    label
+  };
+}
+
+export function dataSubmitterCellData({dataset, label='dataSubmitterCellData'}) {
+  const dataSubmitter = findPropertyValue(dataset, 'Data Depositor');
+  return {
+    data: dataSubmitter,
+    id: dataset.dataSetId,
+    cellStyle: { width: styles.cellWidths.dataSubmitter },
+    label
+  };
+}
+
+export function datasetNameCellData({dataset, label='datasetNameCellData'}) {
+  const datasetName = findPropertyValue(dataset, 'Dataset Name');
+  return {
+    data: div({
+      style: {
+        display: 'flex',
+        alignItems: 'center',
+      }
+    }, [h(Link, {to: `dataset_registration/${dataset.dataSetId}`}, [datasetName])]),
+    value: datasetName,
+    id: dataset.dataSetId,
+    cellStyle: { width: styles.cellWidths.datasetName },
+    label
+  };
+}
+
+export function dataCustodianCellData({dataset, label='dataCustodianCellData'}) {
+  const dataCustodian = findPropertyValue(dataset, 'Data Depositor');
+  return {
+    data: dataCustodian,
+    id: dataset.dataSetId,
+    cellStyle: { width: styles.cellWidths.dataCustodian },
+    label
+  };
+}
+
+export function dataUseCellData({dataset, label='dataUseCellData'}) {
+  getDataUseCodes(dataset);
+  return {
+    data: dataset.codeList,
+    id: dataset.dataSetId,
+    cellStyle: { width: styles.cellWidths.dataUse },
+    label
+  };
+}
+
+export function statusCellData({dataset, label='statusCellData'}) {
+  const updateApprovalStatus = async (status) => {
+    await DAC.updateApprovalStatus(dataset.dacId, dataset.dataSetId, status);
+  };
+
+  // todo: update isRendered/ disabldisabledfor each
+  const dacAccepted = div({
+    style: {color: '#1ea371', fontWeight: 'bold'}
+  },[
+    span('ACCEPTED'),
+    h(Link,{
+      style: {marginLeft: '15px'},
+      id: `${dataset.dataSetId}_edit`,
+      className: 'glyphicon glyphicon-pencil',
+      isRendered: true,
+      to: `dataset_registration/${dataset.dataSetId}`
+    })
+  ]);
+
+  const dacRejected = div({
+    style: {color: '#000000', fontWeight: 'bold'}
+  },[span('REJECTED')]);
+
+  const approvalActions = div({}, [
+    button({
+      id: 'btn_approveDataset',
+      isRendered: true,
+      disabled: false,
+      onClick: () => updateApprovalStatus(true),
+      className: 'btn-primary-dac-datasets'
+    }, ['YES']),
+    button({
+      id: 'btn_rejectDataset',
+      isRendered: true,
+      disabled: false,
+      onClick: () => updateApprovalStatus(false),
```

Comment:
Consider what the grid looks like after the user clicks Yes or No.
When I clicked "No" I would have expected the status in that cell to update after the spinner was done, but it didn't.  Reloading the page caused the page to display the correct value, but that's a hard interruption of my workflow to see the outcome.